nix/patches/0002-add-shebang-to-cwe-checher-to-ida.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/cwe_checker_to_ida.py b/cwe_checker_to_ida.py
+index 50035f3..e3f4d3a 100644
+--- a/cwe_checker_to_ida.py
++++ b/cwe_checker_to_ida.py
+@@ -1,3 +1,5 @@
++#!/usr/bin/env python
```

Comment:
I used a patch file to add a shebang line to this, but if you want I can apply the changes to the original file.